org.eclipse.winery.topologymodeler.ui/src/app/node/properties-content/properties-content.component.ts

```diff
@@ -68,9 +66,9 @@ export class PropertiesContentComponent implements OnInit, OnChanges, OnDestroy
     ngOnInit() {
 
         if (this.currentNodeData.nodeTemplate.properties) {
-            console.log(this.currentNodeData);
             try {
                 const currentProperties = this.currentNodeData.nodeTemplate.properties;
+                console.log(this.currentNodeData.propertyDefinitionType);
                 if (this.currentNodeData.propertyDefinitionType === 'KV') {
```

Comment:
Could you use a string enum for this?
See https://blog.mariusschulz.com/2017/10/27/typescript-2-4-string-enums for a how to.